uitools/import/Esri/ArcGISRuntime/Toolkit/CoordinateConversion.qml

```diff
@@ -65,10 +65,12 @@ Pane {
     property var inputFormat: CoordinateConversionResult { }
 
     /*!
-      \qmlproperty CoordinateConversionController controller.
-      \brief the Controller handles connections writing/reading to the GeoView,
+      \qmlproperty CoordinateConversionController controller
+      \brief The Controller handles connections writing/reading to the GeoView,
       and maintaining our list of textual representations of a single point
       in multiple formats.
+
+      A QML default is provided \l CoordinateConversionController and a CPP is provided \l{Esri::ArcGISRuntime::Toolkit::CoordinateConversionController}{CoordinateConversionController}.
```

Comment:
English is slightly awkward, perhaps?

```text
Tye QML controller is documeneted \l{CoordinateConversionController}{here} and the CPP controller is documented \l{Esri::ArcGISRuntime::Toolkit::CoordinateConversionController}{here}.
```

Applies here and elsewhere.

Looping @ldanzinger for a doc review.

Reply:
Agreed, wording is a bit unclear. I like Neil's suggestion

Reply:
Ok, I will adopt this solution then. Let's remember for next controllers, I think is good for the user to be able to get a link to both the controllers, since the type is `var` and not any specific controller type.